packages/astro/src/integrations.ts

```diff
@@ -47,7 +49,14 @@ export function extraIntegrations() {
     mdx(),
     UnoCSS({
       configDeps: ['./theme.ts'],
-      injectReset: true,
+      injectReset: createRequire(root).resolve('@unocss/reset/tailwind.css'),
```

Comment:
Oh I'm curious: why is the root needed? Is it because `@unocss/reset` is not a dependency of `@tutorialkit/astro`? 

What about we add it as a dependency?

Or we could also keep the `true` value.

Reply:
> Is it because `@unocss/reset` is not a dependency of `@tutorialkit/astro`?

It is a dependency of `@tutorialkit/astro`. 

When `injectReset: true` is used I'm seeing following error:

![image](https://github.com/user-attachments/assets/eede2447-a679-4e21-a417-6b64870ea20a)

Reply:
Oh that is strange, could that be a bug in the unocss astro integration?

Reply:
This is also mentioned on UnoCSS:

- https://github.com/unocss/unocss/issues/2342#issuecomment-1463455572

Now that we don't require end-users (or template) to add `@unocss/reset` as their dependency, its likely causing a hoisting issue here, like mentioned on the issue. Using `require.resolve` solves this.

Reply:
Gotcha, thanks for the clarification!